book/website/reproducible-research/licensing/licensing-software.md

```diff
@@ -1,412 +1,103 @@
+---
+Licenses can either be Free or Propietary, with Free Licenses further classified as Copyleft or Permissive.
+```
 
-<table>
-    <thead>
-        <tr>
-            <th colspan="3">Free</th>
-            <th rowspan="3">Proprietary</th>
-        </tr>
-        <tr>
-            <th colspan="2">Copyleft</th>
-            <th rowspan="2">Permissive</th>
-        </tr>
-        <tr>
-            <th>Strong</th>
-            <th>Weak</th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr>
-        <td>GPL<sup>1</sup> CDDL<sup>2</sup></td>
-        <td>LGPL<sup>3</sup> MPL<sup>4</sup></td>
-        <td>BSD MIT<sup>5</sup> Apache</td>
-            <td>Research-only No&nbsp;copying No&nbsp;modification</td>
-        </tr>
-    </tbody>
-</table>
+(rr-licensing-software-free)=
+## Free Software
 
-<sup>1</sup>GNU General Public License
-<sup>2</sup>CDDL Common Development and Distribution License
-<sup>3</sup>GNU Lesser General Public License
-<sup>4</sup>MPL Mozilla Public License
-<sup>5</sup>BSD MIT Berkeley Software Distribution Massachusetts Institute
+Software licenses are either free or proprietary. 
+Free software comes with license terms that give you ([as defined by GNU](https://www.gnu.org/philosophy/free-sw.html)):
 
-## Free software
-
-The main distinction among software licenses is between Free software and proprietary software.
-Free software come with license terms that give you ([as defined by GNU](https://www.gnu.org/philosophy/free-sw.html))
-
-* freedom 0: The freedom to run the program as you wish, for any purpose.
-* freedom 1: The freedom to study how the program works, and change it so it does your computing as you wish.
+* _Freedom 1_: The freedom to run the program as you wish, for any purpose.
+* _Freedom 2_: The freedom to study how the program works, and change it, so it does your computing as you wish.
 Access to the source code is a precondition for this.
-* freedom 2: The freedom to redistribute copies so you can help others in your community.
-* freedom 3: The freedom to distribute copies of your modified versions to others.
-By doing this you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
+* _Freedom 3_: The freedom to redistribute copies so you can help others in your community.
+* _Freedom 4_: The freedom to distribute copies of your modified versions to others.
+By doing this, you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
 
-These four freedoms together effectively neutralize copyright: **freedoms 1 and 3** let you create derivative works, and **freedoms 2 and 3** let you make copies.
+These four freedoms together effectively neutralize copyright: **freedoms 1 and 3** let you create derivative works, and **freedoms 2 and 4** let you make copies.
```

Comment:
Thanks @martintoreilly for catching this! I remember being confused with the wording here and I also didn't check the linked source.

I prefer your summary over the original -- I am making another commit to revert the numbering back to the 0-4, and including your summary, but with the original numbering.

> @paulowoicho @malvikasharan I've had a go at making the two software licencing tables with formatted HTML. I think this will be cleaner than using an image plus the source to build it. Let me know what you think.
> 
> [Edit: Potentially the table text could be a bit smaller than the main text?]

I also really like how the tables have turned out with your approach. As you suggested in #1467, It may mean that we might also now recommend this table style book-wide for consistency's sake.

Thanks so much for your review on my PR!

Reply:
> I think some readers may confuse the table captions with the main text. 

I've reduced the space between the table caption text and the footnote text at the bottom of the table. Does this make it bind with the table and separate enough from the main text? If not we can also look at making changes to font size, colour or style (e.g. italic).

>As you suggested in #1467, It may mean that we might also now recommend this table style book-wide for consistency's sake.

I haven't figured out how to set styles book wide yet, but we should definitely pick a style and apply it consistently. On that front, I've also removed the alternate row shading from the table style, following the advice from [this article](https://alistapart.com/article/web-typography-tables/#section4) that quotes Tufte. I've left the shading on the row and column headers as I think they help separate these from the data with the multi-row complex column headers, but I'm up for a discussion on what the standard style should be.

@malvikasharan Perhaps best to leave standardising table style across the book to another PR so we can discuss with the community but not block the content changes in this PR or delay the improvement to the readability for these particular tables?

Reply:
> These four freedoms together effectively neutralize copyright: **freedoms 1 and 3** let you use the original software and share it with others, and freedoms **2 and 4** let you create derivative works based on the software and share these with others.

I think the numberings got a bit garbled in the translation back to 0-3 numbering. I've made a commit to group the usage and modification rights as follows, keeping your style changes..

>These four freedoms together effectively neutralize copyright; *Freedoms 0* and *2* let you use the original software and share it with others. *Freedoms 1* and *3* let you create derivative works based on the software and share these with others.